packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -259,6 +259,18 @@ export function isCompatVersionBelowMinVersion(minVersion: string, config: Compa
 	return semver.compare(compatVersion, minReqVersion) < 0;
 }
 
+function genCompatConfig(createVersion: string, loadVersion: string): CompatConfig {
```

Comment:
nit: Would be nice to add a comment describing this function so we know its a helper for `genCrossVersionCompatConfig`